Can't daemonize lobster process

Hello,

I was poking around with lobster/workqueue and ran into the following issue. I'm using the current git `7b0e3da97e73ee5e405c288554bce903dae91127` on top of `CMSSW_8_0_26_patch1` on Centos6.

It appears that `lobster process` without the `--foreground` option balks because there are open filehandles to log files:
```
2017-02-09 16:19:13 [INFO] lobster: saving additional log for configure to /scratch/meloam/lobster_test_20170209_1619/configure.log                                           
2017-02-09 16:19:13 [INFO] lobster.core: saving stderr and stdout to /scratch/meloam/lobster_test_20170209_1619/process.err
2017-02-09 16:19:13 [ERROR] lobster.core: cannot daemonize due to open files       
2017-02-09 16:19:13 [ERROR] lobster.core: open file: /gpfs21/scratch/meloam/lobster_test_20170209_1619/process.log
2017-02-09 16:19:13 [ERROR] lobster.core: open file: /gpfs21/scratch/meloam/lobster_test_20170209_1619/process_debug.log
2017-02-09 16:19:13 [ERROR] lobster.core: open file: /gpfs21/scratch/meloam/lobster_test_20170209_1619/configure.log
2017-02-09 16:19:13 [ERROR] lobster.core: open file: /gpfs21/scratch/meloam/lobster_test_20170209_1619/process.err
```

Running with the `--foreground` option gets a bit further, but I've not sorted out if that is user error yet :)

--- a/lobster/commands/process.py
+++ b/lobster/commands/process.py
@@ -139,6 +139,7 @@ class Process(Command, util.Timing):
 
         process = psutil.Process()
         preserved = [f.name for f in args.preserve]
+        preserved += [os.path.realpath(os.path.abspath(f)) for f in preserved]
         openfiles = [f for f in process.open_files() if f.path not in preserved]
         openconns = process.connections()
 

